butterknife/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="butterknife"/>
+<manifest package="butterknife.core"/>
```

Comment:
This was a side effect of upgrading the AGP version, it looks like single word package names are not allowed anymore. But I don't think it has any impact.

Reply:
Blocked by https://issuetracker.google.com/issues/73450622

Reply:
What is the impact of the change of the manifest ?